### PR TITLE
feat: add hidehistory to add members

### PR DESCRIPTION
**Pull request by peterdeme**
- Add hide history to add members
- Add reviewdog

### CODE REVIEW

File: src/stream-chat-net/IChannel.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public interface IChannel
         Task<TruncateResponse> Truncate();
         Task<TruncateResponse> Truncate(TruncateOptions truncateOptions);
 
-        Task AddMembers(IEnumerable<string> userIDs, MessageInput msg = null);
+        Task AddMembers(IEnumerable<string> userIDs, MessageInput msg = null, bool hideHistory = false);
```
Comment (ferhatelmas) [inline src/stream-chat-net/IChannel.cs:31]: this looks fine but it would be better to introduce an options object to make it extensible

Comment (peterdeme) [reply src/stream-chat-net/IChannel.cs:31]: agreed, its a breaking change though

Comment (ferhatelmas) [reply src/stream-chat-net/IChannel.cs:31]: maybe limit it to last param and keep others same

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**